indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -241,33 +242,18 @@ public Response isLeader()
     }
   }
 
-  @Deprecated
-  @POST
-  @Path("/lockedIntervals")
-  @Produces(MediaType.APPLICATION_JSON)
-  @ResourceFilters(StateResourceFilter.class)
-  public Response getDatasourceLockedIntervals(Map<String, Integer> minTaskPriority)
-  {
-    if (minTaskPriority == null || minTaskPriority.isEmpty()) {
-      return Response.status(Status.BAD_REQUEST).entity("No Datasource provided").build();
-    }
-
-    // Build the response
-    return Response.ok(taskQueryTool.getLockedIntervals(minTaskPriority)).build();
-  }
-
   @POST
-  @Path("/lockedIntervals/v2")
```

Comment:
Let's retain this API as it is not deprecated yet. We can mark it deprecated as a part of the changes in this PR.

Reply:
Done